### PR TITLE
Use dart files as templates because dart lack assets

### PR DESCRIPTION
**Pull request by luanpotter**

### CODE REVIEW

File: test/_inc.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ function clean {
 }
 
 function ignite {
-  dart ../../bin/ignite.dart "$@" 3>&-
+  dart --no-sound-null-safety ../../bin/ignite.dart "$@" 3>&-
```
Comment (luanpotter) [inline test/_inc.sh:12]: sadly one of our deps, prompts, is still not migrated, but there is a PR up for it
https://github.com/thosakwe/prompts

File: lib/main.dart [hunk 2]
```diff
@@ -62,7 +62,7 @@ void mainCommand(List<String> args) async {
 
   final command = results.command;
   if (command?.name == 'create') {
-    await createCommand(command);
+    await createCommand(command!);
```
Comment (luanpotter) [inline lib/main.dart:65]: dart is not smart enough to cast it :/

File: lib/templates/simple/simple_pubspec_yaml.template.dart [hunk 3]
```diff
@@ -0,0 +1,28 @@
+String data({
+  required String name,
+  required String description,
+  required String version,
+  required String flameVersion,
+}) =>
```
Comment (luanpotter) [inline lib/templates/simple/simple_pubspec_yaml.template.dart:6]: not perfect but it's what we can do

Comment (luanpotter) [reply lib/templates/simple/simple_pubspec_yaml.template.dart:6]: btw I stole this idea from how [dcli](https://bsutton.gitbook.io/dcli/dcli-api/assets) solves this issue

**Review by erickzanardo — APPROVED** [review]

**Comment by erickzanardo** [comment]
we con do this as a follow up, but we should start to write tests for this. We could use the test suits for the flutter tool as an idea of how we can test this.

**Comment by luanpotter** [comment]
I have tests for it.

**Pull request merged**